DEVELOPMENT.md

```diff
@@ -0,0 +1,215 @@
+Development
```

Comment:
If we're going to use the archetype, then the project README should just link to the archetype DEVELOPMENT and CONTRIBUTING. See Victory for an example: https://github.com/FormidableLabs/victory#development